packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -271,4 +235,32 @@ export default class EthereumAnchorService extends AnchorService {
         return providers.getDefaultProvider(ethNetwork.network)
     }
 
+    private processRemoteResponse(cidDoc: CidDoc, json: any): boolean {
+        switch (json.status) {
+            case "PENDING": {
```

Comment:
While we're at it, can we explicitly combine the PENDING and PROCESSING cases since the logic is the same?

Reply:
This is intentional in anticipation of proper enum-based handling as in https://github.com/ceramicnetwork/js-ceramic/pull/860/files#diff-7920408ff457031025a51c6c64241f297dc91febb1a810e724475548c154c781R251